sdk/tables/data-tables/review/data-tables.api.md

```diff
@@ -315,20 +316,7 @@ export interface TableSetAccessPolicyHeaders {
 }
 
 // @public
-export class TablesSharedKeyCredential implements TablesSharedKeyCredentialLike {
-    constructor(accountName: string, accountKey: string);
-    readonly accountName: string;
-    computeHMACSHA256(stringToSign: string): string;
-}
-
-// @public
-export interface TablesSharedKeyCredentialLike {
-    accountName: string;
-    computeHMACSHA256: (stringToSign: string) => string;
-}
-
-// @public
-export function tablesSharedKeyCredentialPolicy(credential: TablesSharedKeyCredentialLike): PipelinePolicy;
+export function tablesNamedKeyCredentialPolicy(credential: NamedKeyCredential): PipelinePolicy;
```

Comment:
Do we mean to export the policy from our package?

Reply:
Nope, I'm removing the export. Good catch!